frontend/app/components/contextual-alert.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react';
+
+export interface ContextualAlertProps {
+  message_heading: string;
+  children: ReactNode;
+  type: 'warning' | 'success' | 'danger' | 'info';
+}
+
+export function ContextualAlert(props: ContextualAlertProps) {
+  const { message_heading, children, type } = props;
+
+  const alert_color = type === 'warning' ? 'border-l-amber-700' : type === 'danger' ? 'border-l-red-700' : type === 'info' ? 'border-l-cyan-700' : 'border-l-green-700';
+
+  return (
+    <div id="contextual-alert-id" className={`relative min-w-[290px] pl-[16px] sm:pl-[24px]`}>
+      <div className="absolute left-1.5 top-3 bg-white py-1 sm:left-3.5">
+        <img id="image-id" src={`/assets/${type}_img.svg`} alt="" />
+      </div>
+      <div className={`overflow-auto border-l-4 ${alert_color} py-2.5 pl-[24px] leading-8`}>
+        <h3 className="mb-1 text-xl font-semibold">{message_heading}</h3>
+        <div className="font-body ml-0.5 text-lg">{children}</div>
```

Comment:
I feel we should let the consumer specify the alert body (ie. both heading and children). 

We don't necessarily always want to render an `<h3>` for the heading because if this component were to be used next to an `<h1>`, we would be going from `<h1>` to `<h3>` which would fail validation because of a missing `<h2>`.

Reply:
I agree with @nicholas-ly